policyengine_uk/tests/policy/baseline/gov/dfe/universal_childcare_entitlement/universal_childcare_entitlement.yaml

```diff
@@ -0,0 +1,82 @@
+- name: Child under 3 isn't eligible (person-level)
```

Comment:
unit tests for person-level variables that don't depend on characteristics of other members should be flat (just person-level, no benunit structure)